spec/factory_logger_spec.rb

```diff
@@ -94,4 +94,30 @@
                                             ]
     end
   end
+
+  describe '.descendants' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.